sdcm/cluster.py

```diff
@@ -2789,8 +2845,12 @@ def terminate_node(self, node):
         node.destroy()
 
     def get_db_auth(self):
-        user = self.params.get('authenticator_user', default=None)
-        password = self.params.get('authenticator_password', default=None)
+        if self.params.get('use_ldap_authorization', default=None) and self.ldap_started:
```

Comment:
please remove the `default` param, None is the default if flag wasn't defined

Reply:
ok